standard/standard-library.md

```diff
@@ -464,6 +385,13 @@ A conforming implementation may provide `Task.GetAwaiter()` and `Task<TResult>.G
 ```csharp
 namespace System
 {
+    public class OperationCanceledException : Exception
```

Comment:
Can we keep these declarations in alphabetical order? I think we just need this after FormattableString.

Reply:
@jskeet I moved that type per your suggestion, and I added numerous blank lines throughout C.3 to match the C.2 approach.